### PR TITLE
🖍 [Story interactive] Styling for Image Quizzes and Polls with Missing Images

### PR DESCRIPTION
**Pull request by Brandons42**
Puts a background color in place of a missing image for image quizzes and image polls; the background color is a grey-ish color that depends on the theme of the component

Demos (see second-to-last and third-to-last pages):
https://interactive-img.web.app/missing-img/quiz.html
https://interactive-img.web.app/missing-img/poll.html

/cc @mszylkowski 
/cc @processprocess

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story.css [hunk 1]
```diff
@@ -924,6 +924,7 @@ amp-story .amp-video-eq,
   --i-amphtml-interactive-component-shadow: 0px 4px 12px rgba(60, 64, 67, 0.08), 0px 2px 4px rgba(60, 64, 67, 0.12);
   --i-amphtml-interactive-prompt-clamp: 4 !important;
   --i-amphtml-interactive-results-prompt-margin: 20px !important;
+  --i-amphtml-interactive-missing-img-background: rgba(32, 33, 36, 0.2) !important;
```
Comment (mszylkowski) [inline extensions/amp-story/1.0/amp-story.css:927]: We can call it `--i-amphtml-interactive-placeholder-background` so we can reuse these colors on the results component with the text bubbles

**Review by mszylkowski — APPROVED** [review]

File: extensions/amp-story/1.0/amp-story.css [hunk 2]
```diff
@@ -924,6 +924,7 @@ amp-story .amp-video-eq,
   --i-amphtml-interactive-component-shadow: 0px 4px 12px rgba(60, 64, 67, 0.08), 0px 2px 4px rgba(60, 64, 67, 0.12);
   --i-amphtml-interactive-prompt-clamp: 4 !important;
   --i-amphtml-interactive-results-prompt-margin: 20px !important;
+  --i-amphtml-interactive-placeholder-background: rgba(32, 33, 36, 0.2) !important;
```
Comment (processprocess) [inline extensions/amp-story/1.0/amp-story.css:927]: nit: This might be more descriptive as:
`--i-amphtml-interactive-image-placeholder-background`

**Review by processprocess — APPROVED** [review]
Approving with an optional nit

File: extensions/amp-story/1.0/amp-story.css (hunk 2 above)
Comment (processprocess) [reply extensions/amp-story/1.0/amp-story.css:927]: As discussed in chat, we'll keep this as-is since it might be used for other contexts.

**Pull request merged**